source/sync/data-model/update-schema.txt

```diff
@@ -26,443 +21,518 @@ Update Your Data Model
-.. with the new schema. 
+Changing a property's name on both the server-side schema and the client-side 
+object model is a breaking change. However, some SDKs offer an API to 
+map a new property name to an existing name in the schema. This allows you to 
+rename a property on the client but not change the property name on the server. 
+In this way, you avoid triggering a migration. Property name 
+mapping is supported in the following SDKs:
 
-.. If you choose to change the property's status instead of using the partner collection 
-.. strategy, you must terminate sync, manually update the schema, 
-.. and re-enable sync. In addition, your client 
-.. application must perform a :ref:`client reset <client-resets>` to restore Sync.
-.. In the default client reset mode, the client attempts to recover any 
-.. unsynced changes before resetting.
+- :ref:`Flutter SDK - Remap a Property <flutter-map-property>`
+- :ref:`Java SDK - Rename a Field <java-rename-field>`
+- :ref:`Kotlin SDK - Remap a Property <kotlin-remap-a-property>`
+- :ref:`.NET SDK - Rename a Property <dotnet-rename-property>`
+- :ref:`Node.js SDK - Remap a Property <node-remap-a-property>`
+- :ref:`React Native SDK - Remap a Property <react-native-remap-a-property>`
+- :ref:`Swift SDK - Remap a Property <swift-rename-property>`
+
+.. warning:: Update existing documents
+
+   If you rename a property in the server-side schema, you must 
+   update existing documents with that new property name or they do not sync 
+   to the client. This may give client users the impression that the data 
+   has been lost.
+
+If name mapping is not an option, consider implementing a  
+partner collection strategy, in which 
+you keep the existing collection and schema, and create a new collection 
+with the new schema. This strategy is described in the :ref:`schema-partner_collection-strategy` section on this page.
 
-.. .. note::
+If you choose to change the property's name, you must terminate sync, manually update the schema, 
+and re-enable sync. In addition, your client 
+application must perform a :ref:`client reset <client-resets>` to restore Sync.
+In the default client reset mode, the client attempts to recover any 
+unsynced changes before resetting.
+
+When you terminate and re-enable sync, you must also update existing Atlas 
+documents to enable them to Sync with your client applications. 
+Without this additional handling, those documents do not Sync and it may 
+appear in the client that the data has been lost. You could resolve this issue 
+in two ways:
+
+- Change the old field name on each document to match the new schema
+- Add a new field to each document with a name that matches the new schema, 
+  and copy the value from the old field into it
+
+After you've made these changes, the appropriate documents sync to the client 
+application.
+
+.. _schema-update-change-property-type-but-keep-name:
+
+Change a Property Type but Keep the Name
```

Comment:
Probably out of scope for this ticket, but it would be nice to add some guidance on how to change a type using schema versions. It's a bit less painful with the new feature, so I think it would be helpful to expose that to users